index.html

```diff
@@ -0,0 +1,74 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="./style.css">
+  <title>project-page</title>
+</head>
+<body>
+<header>
+  <h1> MY RECENT WORK</h1>
+</header>
+<hr class="border">
+  <section class="multi-post">
+<div class="multi-story-1">
+  <img src="./image/blank-img.png" alt="image-placeholder">
+<h2>Multi-Post Stories</h2>
+<p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.</p>
+<div class="list">
+  <ul>
+    <li class="css">css</li>
+    <li class="html">html</li>
+ <li class="bootstrap">bootstrap</li>
+     <li class="ruby"> ruby </li>
+   </ul>
+</div>
+<input type="button" value="See Project">
+</div>
+<div class="multi-story-2">
+<img src="./image/blank-img.png" alt="image-placeholder">
+<h2>Multi-Post Stories</h2>
+<p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.</p>
+<div class="list">
+  <ul>
+    <li class="css">css</li>
+    <li class="html">html</li>
+ <li class="bootstrap">bootstrap</li>
+     <li class="ruby"> ruby </li>
+   </ul>
+</div>
+<input type="button" value="See Project">
+</div>
+<div class="multi-story-3">
+  <img src="./image/blank-img.png" alt="image-placeholder">
+<h2>Multi-Post Stories</h2>
+<p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.</p>
+<div class="list">
+  <ul>
+    <li class="css">css</li>
+    <li class="html">html</li>
+ <li class="bootstrap">bootstrap</li>
+     <li class="ruby"> ruby </li>
+   </ul>
```

Comment:
- [ ] The `font` for this elements should be `Font: Inter`. It will be good to use the exact font so that your template can resemble the given design.